src/cascadia/TerminalApp/defaults.json

```diff
@@ -233,6 +233,7 @@
         { "command": "switchToTab7", "keys": ["ctrl+alt+8"] },
         { "command": "switchToTab8", "keys": ["ctrl+alt+9"] },
         { "command": "decreaseFontSize", "keys": ["ctrl+-"] },
-        { "command": "increaseFontSize", "keys": ["ctrl+="] }
+        { "command": "increaseFontSize", "keys": ["ctrl+="] },
+        { "command": "toggleFullscreen", "keys": ["alt+enter"] }

```

Comment:
Does `alt+enter` cause a conflict with `altgr` on German (and other such) keyboards?
Perhaps `f11` instead?

Reply:
I honestly have no idea - summoning resident alt-gr expert @lhecker

Reply:
> summoning resident alt-gr expert

😂
I'm not aware of <kbd>Alt</kbd><kbd>Enter</kbd> generating anything related to <kbd>AltGr</kbd> or vice versa. For now we're safe either way though, since the shortcut handler is not invoked at all if <kbd>AltGr</kbd> is being pressed (due to it not support a distinction between left and right <kbd>Alt</kbd>).

May I suggest using <kbd>F11</kbd>either way though?
It's what Gnome Terminal, Firefox, Chrome and probably quite a few others already use. 🙂

Reply:
![both-is-good](https://user-images.githubusercontent.com/18356694/68055924-d55b6300-fcbf-11e9-882b-214df8725ded.gif)